config/projects/omnibus-toolchain.rb

```diff
@@ -20,14 +20,20 @@
 license "Apache-2.0"
 license_file "LICENSE"
 
-install_dir "/opt/omnibus-toolchain"
+if windows?
+  install_dir  "#{default_root}/chef/#{name}"
```

Comment:
Yeah, I am not sure why it was set that way... Probably copy-pasta. If it's at all confusing I don't mind changing it --  In my mind I saw this as the company name and stuff inside being the products.